lambda/test/handler.test.ts

```diff
@@ -0,0 +1,88 @@
+import { generateMessage, getFormattedInstanceList } from "../src/handler";
+import { Instance } from "../src/interfaces";
+
+describe("handler", function () {
+  describe("generateMessage", function () {
+    it("should return a list of instances with missing tags", function () {
+      const instanceList = getFormattedInstanceList(([
+        {
+          tags: { App: "app1", Stack: "stack1" },
+          instanceName: "instance1",
+          meta: { href: "href1" },
+        },
+        {
+          tags: { App: "app2", Stack: "stack2" },
+          instanceName: "instance2",
+          meta: { href: "href2" },
+        },
+      ] as unknown) as Instance[]);
```

Comment:
Ooh this is interesting! Could we add a comment against this casting?

Reply:
Sounds good! I'm struggling to come up with a good comment though, do you have an example? Is this even the best approach for this?

Reply:
The `Partial<Instance>` specification looks like a better solution to `as unknown as Instance[]`; it looks like there are a couple more instances of `as unknown` that we can replace with this too.

Reply:
Thanks! Didn't realise I missed the other ones, they've now been updated too.